src/customExpressHttpContext.ts

```diff
@@ -3,7 +3,7 @@ import uuid from "uuid";
 
 const REQUEST_ID = "requestId";
 
-export const setRequestId = () => {
+export const setRequestId = (value? : string) => {
     return httpContext.set(REQUEST_ID, uuid.v1());
```

Comment:
the reason it maybe isnt working is because you are calling v1 function, i havent tried it so its just a wild guess, but on a side note, i suggest to use v4 (random) because v1 is predictable

Reply:
hmm, I'll check that